relayer/workers/parachaincommitmentrelayer/beefy-listener.go

```diff
@@ -76,26 +76,27 @@ func (li *BeefyListener) Start(ctx context.Context, eg *errgroup.Group) error {
 
 	eg.Go(func() error {
 
-		verifiedBeefyBlockNumber, verifiedBeefyBlockHash, err := li.fetchLatestVerifiedBeefyBlock(ctx)
+		relayBlockNumber, relayBlockHash, err := li.fetchLatestVerifiedBeefyBlock(ctx)
```

Comment:
prefer to keep these variables named as is. relay is a terribly ambiguous word.

Reply:
Sure, I agree regarding beefy vs relay and will make the change. Though I'd drop the `verified` prefix. The signal to noise ratio of `verifiedBeefyBlockNumber` and `verifiedParaBlockNumber` is quite low, so dropping `verified` makes it easier to discern between the two things.

Reply:
fixed in latest upload